aconcli/cmd/generate.go

```diff
@@ -18,12 +18,13 @@ import (
 )
 
 var generateCmd = &cobra.Command{
-	Use:   "generate DOCKER_IMAGE",
-	Short: "Generate a manifest and commit file system layers to ACON repository",
+	Use:   "generate <Docker image>",
```

Comment:
Here `<Docker image>` has the 1st letter capitalized but other (e.g., `<manifest file>`) place holders are all in lower case. It'd be better to have a consistent style.

Also, it looks like `git` style placeholders (lower case phrases surrounded by angle brackets) are being used here. If so, please use hyphens (`-`) in place of spaces (` `) for separating words.

Reply:
resolved